.env

```diff
@@ -1,3 +1,6 @@
-DB_NAME=squid
-DB_PORT=23798
-GQL_PORT=4350
\ No newline at end of file
+INDEXER_ENV=featurenet
+DB_PORT=5433
+GATEWAY_PROTOCOL=http
+GATEWAY_HOST=127.0.0.1
+GATEWAY_PORT=8000
+GQL_PORT=4355
```

Comment:
I hate to say it, but everyone always did the same to me :smiling_face_with_tear: 

nit: no newline here

Reply:
o tempora o mores :raised_hands: Fixed